src/Uno.UWP/Microsoft/UI/DisplayId.cs

```diff
@@ -13,20 +13,13 @@ public DisplayId(ulong _Value)
 			=> x.Value == y.Value;
 
 		public static bool operator !=(DisplayId x, DisplayId y)
-			=> !(x == y);
+			=> x.Value != y.Value;
 
 		public bool Equals(DisplayId other)
-			=> this == other;
+			=> Value == other.Value;
 
 		public override bool Equals(object obj)
-		{
-			if (obj is DisplayId)
-			{
-				DisplayId y = (DisplayId)obj;
-				return this == y;
-			}
-			return false;
-		}
+			=> (obj is DisplayId y) && Equals(y);
```

Comment:
I think the parentheses are unnecessary.

```suggestion
			=> obj is DisplayId y && Equals(y);
```